book/03-git-branching/sections/nutshell.asc

```diff
@@ -1,81 +1,60 @@
 [[_git_branches_overview]]
-=== Branches in a Nutshell
+=== Mga Branch sa Maikling Salita
 
-To really understand the way Git does branching, we need to take a step back and examine how Git stores its data.
+Upang talagang maintindihan ang paraan ng Git sa paggawa ng pag-branch, kailangan nating umatras at suriin kung paano nag-iimbak ng data ang Git.
 
-As you may remember from <<_getting_started#_getting_started>>, Git doesn't store data as a series of changesets or differences, but instead as a series of snapshots.
+Kung iyong iyong natatandaan mula sa <<_getting_started#_getting_started>>, ang Git ay hindi nag-iimbak ng data bilang isang serye ng mga changeset o mga kaibahan, ngunit sa halip ay isang serye ng mga snapshot.
 
-When you make a commit, Git stores a commit object that contains a pointer to the snapshot of the content you staged.
-This object also contains the author's name and email, the message that you typed, and pointers to the commit or commits that directly came before this commit (its parent or parents): zero parents for the initial commit, one parent for a normal commit, and multiple parents for a commit that results from a merge of two or more branches.
+Kapag ikaw ay gumawa ng isang commit, ang Git ay nag-iimbak ng isang commit na object na naglalaman ng isang pointer sa snapshot ng nilalaman na iyong na-stage. Ang object na ito ay naglalaman din ng pangalan ng may-akda at email, ang mensahe na iyong na-type, at mga pointer sa commit o mga commit na direktang dumating bago ang commit na ito (magulang nito o mga magulang): zero na mga magulang para sa paunang commit, isang magulang para sa isang normal na commit, at maramihang mga magulang para sa isang commit na nagreresulta mula sa isang pag-merge ng dalawa o higit pang mga branch.
```

Comment:
Please change `Ang object na ito ay naglalaman din ng pangalan ng may-akda at email` to `Ang object na ito ay naglalaman din ng pangalan at email ng may-akda`

Reply:
Thank you for pointing that out. Will update now.